include/clink/feature/one_hot_encoder.h

```diff
@@ -0,0 +1,66 @@
+/*
+ * Copyright 2021 The Clink Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#ifndef CLINK_FEATURE_ONE_HOT_ENCODER_H_
+#define CLINK_FEATURE_ONE_HOT_ENCODER_H_
+
+#include "clink/feature/proto/one_hot_encoder.pb.h"
+#include "clink/linalg/sparse_vector.h"
+
+namespace clink {
+
+// A Model which encodes data into one-hot format.
+class OneHotEncoderModel {
```

Comment:
Could we have C++ test for `OneHotEncoderModel`? This could help provide example of adding C++ test in Clink. And this could help verify that the class can be used in pure C++ setup directly.

Reply:
OK. I'll add tests.